src/main/java/edu/kis/powp/jobs2d/drivers/adapter/BasicLineDrawerAdapter.java

```diff
@@ -23,11 +25,12 @@ public void setPosition(int x, int y) {
 
 	@Override
 	public void operateTo(int x, int y) {
-		ILine line = LineFactory.getBasicLine();
+		ILine line = new ParametrizedLineAdapter(LineFactory.getBasicLine());
```

Comment:
How ParametrizedLineAdapter can be used here?
Is wrapping line with ParametrizedLineAdapter here necessary? If not should be removed.